test/test_torch.py

```diff
@@ -5604,10 +5604,11 @@ def to_np(value):
                 actual = base.pow(exponent)
                 self.assertEqual(actual, expected.to(actual), allow_inf=True)
 
-                actual = base.clone()
-                actual2 = actual.pow_(exponent)
-                self.assertEqual(actual, expected.to(actual), allow_inf=True)
-                self.assertEqual(actual2, expected.to(actual), allow_inf=True)
+                if torch.can_cast(torch.result_type(base, exponent), base.dtype):
```

Comment:
@ngimel and I recommend running the other code path (that you can't cast) and asserting it throws an exception

Reply:
added